src/detectors/transpilers/html/parser.ts

```diff
@@ -0,0 +1,59 @@
+import type {ReadStream} from "node:fs";
+import {Detail, SaxEventType, SAXParser, Tag as SaxTag} from "sax-wasm";
+import {finished} from "node:stream/promises";
+import fs from "node:fs/promises";
+import {createRequire} from "node:module";
+const require = createRequire(import.meta.url);
+
+let saxWasmBuffer: Buffer;
+async function initSaxWasm() {
+	if (!saxWasmBuffer) {
+		const saxPath = require.resolve("sax-wasm/lib/sax-wasm.wasm");
+		saxWasmBuffer = await fs.readFile(saxPath);
+	}
+
+	return saxWasmBuffer;
+}
+
+async function parseHtml(contentStream: ReadStream, parseHandler: (type: SaxEventType, tag: Detail) => void) {
+	const options = {highWaterMark: 32 * 1024}; // 32k chunks
+	const saxWasmBuffer = await initSaxWasm();
+	const saxParser = new SAXParser(SaxEventType.OpenTag | SaxEventType.CloseTag, options);
```

Comment:
```suggestion
	const saxParser = new SAXParser(SaxEventType.CloseTag, options);
```

Since we are only interested in closing tags, do not request the OpenTag event

Reply:
Yep, thanks! It left as I was uncertain where I could get the internal TextNodes. obviously, they are available, after the close tag